DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -271,7 +271,8 @@
    <version ClassVersion="11" checksum="1239840459"/>
   </class>
 
-  <class name="pat::PackedCandidate" ClassVersion="19">
+  <class name="pat::PackedCandidate" ClassVersion="21">
+    <version ClassVersion="21" checksum="663492232"/>
```

Comment:
Hi Bill,

there is ``` <version ClassVersion="21" checksum="1075665714"/> ``` in 75X already
and we have queued a ``` <version ClassVersion="22" checksum="691064527"/> ``` into 74X (to be forward ported to 75X/76X).

So, I think this one will need to be ClassVersion="23"